### PR TITLE
Improve compat suite

### PR DESCRIPTION
**Pull request by turbolent**
- Add a `requirements.txt` that lists all dependencies
- Add instructions how to run the suite

### CODE REVIEW

File: compat/README.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
 # Source Compatibility Suite
 
+## Running
```
Comment (zhangchiqing) [inline compat/README.md:3]: Could you briefly explain what the compatibility suite do?

File: compat/README.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
 # Source Compatibility Suite
```
Comment (turbolent) [inline compat/README.md:1]:
```suggestion
# Source Compatibility Suite

The source compatibility suite prevents regressions and helps understand what impact changes in the language and implementation have on real-world Cadence projects.

The suite contains a [set of repository descriptions](https://github.com/onflow/cadence/tree/master/compat/suite). When the suite is run, the repositories get cloned and checked. The runner can optionally run benchmarks and compare against another commit / branch, producing output in the terminal (pretty), as JSON, or Markdown.

In the future we can integrate this as part of CI, maybe as a periodic job.
```

File: compat/README.md (hunk 1 above)
Comment (turbolent) [reply compat/README.md:9]: Added above, also see #226

**Review by zhangchiqing — APPROVED** [review]
👍

**Pull request merged**